src/webgl/texture.js

```diff
@@ -112,6 +112,13 @@ export const TEXTURE_FORMATS = {
   // [GL.COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL]: {compressed: true, gl1: ETC1}
 };
 
+const V3_TO_V4_OPTIONS_MAP = {
```

Comment:
I am pretty sure the WebGL2 parameters have never been used, since there were other issues with the WebGL2 texture support in v3.

So for compatibility reasons, the answer is no.

We could decide to keep this API, it is a little easier to use than GL ENUMS. Although it would be the only class for which we have our own parameters. What do you think?